.github/workflows/ci.yml

```diff
@@ -345,6 +345,37 @@ jobs:
         run: rush build -o calling
       - name: Tests
         run: rush test -o calling
+      # Not check in the bundles stats if size doesn't change
+      # Stats size in the file is a bit random, so we can't only depend on file itself
+      - name: Check size change
+        id: changescheck
+        if: ${{ matrix.flavor == 'beta' && github.event_name == 'pull_request' }}
+        run: |
+          base_size=$(jq -r ".[0].parsedSize" samples/Calling/bundle-report/report.json)
+          current_size=$(jq -r ".[0].parsedSize" samples/Calling/dist/build/report.json)
+          change=$([ $current_size = $base_size ] && echo "true" || echo "false")
+          echo "::set-output name=hasChanged::${change}"
+      - name: Copy current bundle report json
+        if: ${{ steps.changescheck.outputs.hasChanged == 'true' }}
+        run: cp samples/Calling/dist/build/report.json samples/Calling/bundle-report/
```

Comment:
Why are we replacing the new report with the old one here?

Reply:
Because we want to check the new report in, so that new report will be in the current branch, and ready to be checked in when merged